src/backend/distributed/planner/multi_master_planner.c

```diff
@@ -151,6 +152,58 @@ BuildAggregatePlan(Query *masterQuery, Plan *subPlan)
 }
 
 
+static Plan *
+BuildDistinctPlan(Query *masterQuery, Plan *topLevelPlan)
+{
+	bool distinctClausesHashable = true;
+	List *distinctClauseList = masterQuery->distinctClause;
+	Plan *plan = topLevelPlan;
```

Comment:
I think no need to rename this and introduce another variable name. Shall we call this as subPlan and use it similar to `BuildAggregatePlan`.

Reply:
ok